content/getting-started/index.mdx

```diff
@@ -8,9 +8,11 @@ To use Chakra UI in your project, run the following command in your terminal:
 
 ```bash
 npm i @chakra-ui/react @emotion/react @emotion/styled framer-motion
+```
 
-# or
+or
```

Comment:
I'd remove the "or" and rephrase the sentence above to something like: `To use Chakra UI in your project, run one of the following commands in your terminal:`

Reply:
Thanks for the suggestion! Made changes accordingly